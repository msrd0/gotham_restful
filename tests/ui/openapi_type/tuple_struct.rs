#[macro_use]
extern crate gotham_restful;

#[derive(OpenapiType)]
struct Foo(String);

fn main() {}
