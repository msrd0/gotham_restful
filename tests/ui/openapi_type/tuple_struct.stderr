error: #[derive(OpenapiType)] does not support unnamed fields
 --> $DIR/tuple_struct.rs:5:11
  |
5 | struct Foo(String);
  |           ^^^^^^^^
